inkscape/src/main/assembly/dist/lace_grid.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Grid</_name>
+	<_name>Rectangular Grid</_name>

```

Comment:
In Math it is called a triangle lattice or a rhombic lattice.

The term rectangular grid is not really accurate because 1) the angle is not always 90 degrees (orthogonal grid is similarly incorrect for this reason) 2) the length of the sides is always equal.  

I chose rectangular grid because that is the term used on the wiki page:  https://bobbin-lace.wikispaces.com/grids

Jo Edkins talks about square and hexagonal grids:  http://gwydir.demon.co.uk/jo/lace/grids.htm

Several books (such as Pamela Nottingham’s “The Technique of Bucks Point Lace” just talk about grids without any adjective.

The main purpose is to differentiate between polar grids and regular grids.  I think the most accurate option would be:

    Bobbin Lace
        Grid ... 
        Polar Grid ..

But this may be confusing and will become more confusing if we start to add other kinds of grids (like logarithmic and distorted grids).

I propose:

    Bobbin Lace
        Diamond Grid ... 
        Polar Grid ...

Diamond is a common name for rhombus and is correct for different angles.

From: jo-pol 
Sent: Monday, April 13, 2015 3:28 AM
To: jo-pol/DiBL 
Cc: Veronika Irvine 
Subject: Re: [DiBL] contributor role figured out (#4)

In inkscape/src/main/assembly/dist/lace_grid.inx:

> @@ -1,7 +1,7 @@
>  <?xml version="1.0" encoding="UTF-8"?>
>  <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
>  
> -	<_name>Lace Grid</_name>
> +	<_name>Rectangular Grid</_name>
I would have called it orthogonal, but most lace makers will understand this better.

—
Reply to this email directly or view it on GitHub.